### DISCUSSION TITLE
Need .NET MAUI MSIX icon background and shortcut tile to be transparent!

### DISCUSSION THREAD

**Discussion by Nav-Pandher** [dotnet/maui#22964] [Q&A]
When i have built the MSIX file from VS2022 and click on it to install the application, in the top right hand corner the icon has a background sitting around my icon.

Also when I create a shortcut on the desktop, it also has a tile background.

For example, in the screenshot below, I've used a test icon which has a transparent background, but it shows within a blue square.

![image](https://github.com/dotnet/maui/assets/114393351/4584cc2c-3a59-45ee-b6e4-1d0258237e91)

Then for the shortcut, you can see in the screenshot below that it has a blue tile around the icon. (the black colour is just my desktop background)

![image](https://github.com/dotnet/maui/assets/114393351/d1d99c63-bb7f-44da-8df2-97f1e519a188)

In the start menu and the taskbar, the app shows correctly with a transparent background:

![image](https://github.com/dotnet/maui/assets/114393351/4816a0e8-8fdd-46ae-ae4a-76ace2f00a17)

I need the MSIX installer to just show my icon with no background like many other apps, see the example below:

![image](https://github.com/dotnet/maui/assets/114393351/15e553c8-6f2c-4fe7-a0fe-e30cf1a179f5)

**Comment by malciin** [comment]
https://github.com/dotnet/maui/issues/23133
